src/bootstrap/bootstrap.py

```diff
@@ -395,15 +394,15 @@ class RustBuild(object):
     def __init__(self):
         self.checksums_sha256 = {}
         self.stage0_compiler = None
-        self._download_url = ''
+        self.download_url = ''
         self.build = ''
         self.build_dir = ''
         self.clean = False
         self.config_toml = ''
         self.rust_root = ''
-        self.use_locked_deps = ''
-        self.use_vendored_sources = ''
-        self.verbose = False
+        self.use_locked_deps = False
+        self.use_vendored_sources = False
+        self.verbose = 0
```

Comment:
If the old variable could be kept, it would be better, since this allows keeping the simpler `self.verbose` arguments.
Maybe add `self.verbose_count`?